src/aliceVision/image/io.hpp

```diff
@@ -57,13 +57,15 @@ enum class EImageFileType
  */
 struct ImageReadOptions
 {  
-  ImageReadOptions(EImageColorSpace colorSpace = EImageColorSpace::AUTO, bool useWhiteBalance = true, const oiio::ROI & roi = oiio::ROI()) :
```

Comment:
I think it makes sense to start using builder pattern here, as that would simplify how the read functions can be called.